index.html

```diff
@@ -54,6 +54,21 @@
           Register</button>
       </div>
     </nav>
+    <div class="spicestore">
+      <h1 class="tooltip" onclick="history.go(0);"> spice<span class="word2">store</span>
+      <span class="tooltiptext">Home</span>
```

Comment:
instead of adding separate html tag, you could have used <h1 title="Home">some text</h1>